README.md

```diff
@@ -16,18 +16,44 @@ Blog posts are located in [`./src/pages/blog`](https://github.com/ParalelnaPolis
 
 ## Development
 
+### Prerequisities
+
+- you need [node with npm (node package manager)](https://nodejs.org/en/download/) installed to be able to build and run website locally on you computer. Let's stick with LTS (Long Term Support) version.
+- [git](https://git-scm.com/downloads) is needed to commit new changes; you can find also some [GUI clients](https://git-scm.com/downloads/guis) that may help you with that.
+  - how to work with git? [the simple guide - no deep shit!](http://rogerdudler.github.io/git-guide/)
+- your favourite text editor (humbly recommending [vscode](https://code.visualstudio.com/))
+
+### Installation
+
+In the terminal run:
+
 ```
 git clone git@github.com:ParalelnaPolisKE/web.git
+cd web
 npm install
+```
+
+Create new `.env` file and set correct constants inside (this may not be needed in the future):
+
+```
+cp .env.example .env
```

Comment:
C:\Users\Lukas\web>cp .env.example .env
'cp' is not recognized as an internal or external command,
operable program or batch file.

Reply:
got it - https://stackoverflow.com/questions/21734377/cp-command-prompt-windows-7-not-recognized

"By default, there's no cp in Windows. The equivalent Windows command is copy."